.github/workflows/inspektor-gadget.yml

```diff
@@ -828,6 +834,12 @@ jobs:
         file: inspektor-gadget-${{ github.ref_name }}.yaml
         github-token: ${{ secrets.GITHUB_TOKEN }}
         release-url: ${{ steps.create_release.outputs.upload_url }}
+    - name: Upload checksums file
+      uses: csexton/release-asset-action@v2
+      with:
+        file: inspektor-gadget-${{ github.ref_name }}_checksums.txt
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        release-url: ${{ steps.create_release.outputs.upload_url }}
```

Comment:
Any specific reason to have a single file with all cheksums instead of a file per checksum? 

I don't have any preference, I see both approaches are used in different repos, but I wanted to confirm if there is any specific reason to use one or another way. 

https://github.com/containerd/containerd/releases/tag/v1.6.15 vs https://github.com/containers/podman/releases/tag/v4.3.1